.github/workflows/frontend-setup/action.yml

```diff
@@ -34,3 +34,7 @@ runs:
     - name: Install frontend dependencies
       shell: bash
       run: pnpm install
+
+    - name: Install playwright
+      shell: bash
+      run: npx playwright install --with-deps
```

Comment:
Just have it in the same dependencies install step. What's the need of having this as a separate step?
Also, does `pnpm install` not cover it if it's already in the `package.json`?

Reply:
To run e2e, we need to download this command line tools along with all the browsers, thats what this step does...
https://playwright.dev/docs/ci